### DISCUSSION TITLE
I want to know how to uniform each vehicles travel times as much as possible.

### DISCUSSION THREAD

**Discussion by Yoshikazu1982** [google/or-tools#2827] [Routing (and legacy CP) questions]
Now I will try to calculate VRPTW.
But I want to know that "the goal is to minimize the range of the each vehicles travel time",
not "The goal is to minimize the total travel time of the vehicles." for Guides.

Could you tell me how to minimize the range of each time?
ex) The range means that 20min. if vehicle1 10min., vehicle2 20min., vehicle3 30min.
I want to uniform each vehicles travel times as much as possible.

And, I specify the number of vehicles, but sometimes, vehicles 0min.
I want to check if this  solution is optimal or not.

Thanks!

**Comment by paultrow** [comment]
You can't "minimize the range of each time": the set of travel times for
the vehicles is a vector, and vectors don't have a natural ordering. You
can only minimize a single numeric quantity for all vehicles, such as total
travel time, or the maximum travel time among all vehicles (see
https://developers.google.com/optimization/routing/vrp). I'm guessing the
latter example might be what you're looking for.

**Reply by Yoshikazu1982** [reply]
Thank you for your reply.

I understand the vectors don't have  a natural ordering
and it is impossible to calculate for my goal.

I will try to customize the script using the example!
